src/Components/GameScreen.tsx

```diff
@@ -1,9 +1,20 @@
 import React from "react"
+import Board from "./GameScreenComponents/Board"
+import GameHeader from "./GameScreenComponents/GameHeader";
+import GameFooter from "./GameScreenComponents/GameFooter";
 
-function GameScreen(){
+interface PlayerInfo{
+    playerNames:{Player1Name:string, Player2Name:string}
+}
+
+function GameScreen(props: PlayerInfo){
     return (
-        <div>This is the game screen</div>
+        <div>
+            <GameHeader playerName={props.playerNames.Player1Name}/>
```

Comment:
Just to check, is only player 1's name being passed down to the header a temporary thing which will be changed later?

Reply:
Yes, that was temporary, it'll be updated in the next commits